familyconnections/admin/debug.php

```diff
@@ -0,0 +1,380 @@
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Memory Usage').'</td>
+                                <td>'.formatsize(memory_get_usage()).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Memory Limit').'</td>
+                                <td>'.(ini_get('memory_limit') ? formatsize(ini_get('memory_limit')) : T_('N/A')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Max Upload Size').'</td>
+                                <td>'.(ini_get('upload_max_filesize') ? formatsize(ini_get('upload_max_filesize')) : T_('N/A')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Max Post Size').'</td>
+                                <td>'.(ini_get('post_max_size') ? formatsize(ini_get('post_max_size')) : T_('N/A')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Safe Mode').'</td>
+                                <td>'.(ini_get('safe_mode') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Allow URL fopen').'</td>
+                                <td>'.(ini_get('allow_url_fopen') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Allow URL Include').'</td>
+                                <td>'.(ini_get('allow_url_include') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Display Errors').'</td>
+                                <td>'.(ini_get('display_errors') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Display Startup Errors').'</td>
+                                <td>'.(ini_get('display_startup_errors') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Expose PHP').'</td>
+                                <td>'.(ini_get('expose_php') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Register Globals').'</td>
+                                <td>'.(ini_get('register_globals') ? T_('On') : T_('Off')).'</td>
+                            </tr>
+                            <tr>
+                                <td>'.T_('PHP Max Script Execution Time').'</td>
+                                <td>'.(ini_get('max_execution_time') ? sprintf(T_('%s seconds'), ini_get('max_execution_time')) : T_('N/A')).'</td>
```

Comment:
Please tell me the php file name where the PHP Max Script Execution Time is located..